### PR TITLE
removed activations from fully connected layer 

### PR DESCRIPTION
**Pull request by kchoo1118**
1) removed activations from fully connected layer 
2) lookups were simplified
3) info messages modified

Todo:
1) conv net issue
2) additional layer information

### CODE REVIEW

File: NetKet/Machine/activation_layer.hpp [hunk 1]
```diff
@@ -75,30 +76,25 @@ class Activation : public AbstractLayer<T> {
 
   void InitLookup(const VectorType &v, LookupType &lt,
                   VectorType &output) override {
-    lt.resize(1);
-    lt[0].resize(size_);
+    lt.resize(0);
```
Comment (gcarleo) [inline NetKet/Machine/activation_layer.hpp]: Do you need to resize it at all?

File: Test/Machine/test_machine.py [hunk 2]
```diff
@@ -31,8 +31,7 @@
 layers = [
     nk.layer.FullyConnected(
         input_size=g.n_sites,
-        output_size=40,
-        activation=nk.activation.Lncosh())
+        output_size=40)
```
Comment (gcarleo) [inline Test/Machine/test_machine.py]: Shouldn't you add an activation layer here?

**Comment by kchoo1118** [comment]
also the tutorial is outdated i think

File: Test/Machine/test_machine.py (hunk 2 above)
Comment (gcarleo) [reply Test/Machine/test_machine.py]: yes, but maybe in the next PR we should add checks for all the implemented layers, just to make sure it's all OK!

**Comment by gcarleo** [comment]
> also the tutorial is outdated i think 

yes, that's issue #94 , we will fix that once we finish stabilizing the interface of the various classes. At the moment we are missing just a couple of classes

**Pull request merged**